app/src/main/java/com/taskruler/RetrofitClientInstance.kt

```diff
@@ -5,14 +5,14 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "https://wrytheruc.github.io/"
+    private const val BASE_URL = "https://wrytheruc.github.io/"
 
     val retrofitInstance : Retrofit?
         get() {
             // has this object been created yet?
             if (retrofit == null) {
                 // create it
-                retrofit = retrofit2.Retrofit.Builder()
+                retrofit = Retrofit.Builder()
```

Comment:
Redundant quantifier name. A fully qualified name may not be necessary due to a static import. 
https://tech.okcupid.com/cutting-through-the-static-find-and-fix-issues-the-smart-way-with-android-studio-static-code-fb0bc99d9788